suite-native/module-accounts-management/src/selectors.ts

```diff
@@ -1,42 +1,23 @@
-import { D, pipe } from '@mobily/ts-belt';
-
-import { type NetworkSymbol, getNetworkType, networks } from '@suite-common/wallet-config';
+import { type NetworkSymbol, getNetworkType } from '@suite-common/wallet-config';
 import {
     FeatureFlagsRootState,
     selectIsFeatureFlagEnabled,
     FeatureFlag,
 } from '@suite-native/feature-flags';
 
-const PRODUCTION_SEND_COINS_WHITELIST = pipe(
-    networks,
-    D.filter(
-        network =>
-            network.networkType === 'bitcoin' ||
-            network.networkType === 'ethereum' ||
-            network.networkType === 'ripple',
-    ),
-    D.keys,
-);
-
 export const selectIsNetworkSendFlowEnabled = (
     state: FeatureFlagsRootState,
-    symbol?: NetworkSymbol,
+    networkSymbol?: NetworkSymbol,
 ) => {
-    if (!symbol) return false;
-    const networkType = getNetworkType(symbol);
-
-    if (PRODUCTION_SEND_COINS_WHITELIST.includes(symbol)) return true;
+    if (!networkSymbol) return false;
+    const networkType = getNetworkType(networkSymbol);
 
     const isCardanoSendEnabled = selectIsFeatureFlagEnabled(
         state,
         FeatureFlag.IsCardanoSendEnabled,
     );
 
-    if (isCardanoSendEnabled && networkType === 'cardano') return true;
-
-    const isSolanaSendEnabled = selectIsFeatureFlagEnabled(state, FeatureFlag.IsSolanaSendEnabled);
-
-    if (isSolanaSendEnabled && networkType === 'solana') return true;
+    if (networkType !== 'cardano' || isCardanoSendEnabled) return true;
```

Comment:
send for all the other networkTypes is already enabled, since we will enable sending solana at the same time as we start supporting it.

Reply:
makes sense to simplify that 👍